### DISCUSSION TITLE
Surfing public sphere: A map of daily protests & signs in Iran

### DISCUSSION THREAD

**Discussion by creeptonight** [WeAreMahsaAmini/Core#466] [Ideas]
Over the past few days we observed that cases of harmonic activities, civil disobediences, graffiti, fliers, etc. in the streets of Iran are sometimes overshadowed by the videos of public protests. These smaller, yet extremely important activities are not well reflected and broadcasted across the country. We believe that having a dedicated platform for mapping these activities over the country, and showing them to the public, can lead to an increased pace and balance of spreading the protests. Such a platform can help share the ideas and change the faces of the cities. Moreover, this can encourage those people who are scared of street public protests, to get involved in the more hidden and underground part of the movement.

A group of activists in Iran have proposed creating a platform (here a website) which harmonizes activities countrywide. It can be an interactive map of Iran, where each city or region, shows a badge of notifications indicating the number of daily events (protests, graffiti, art work, fliers, announcements, etc.). By clicking on the notification badge of one region, the map zooms in on the corresponding area, showing the uploaded content (pics and videos) as pins in different areas of the city. 

For a detailed proposal, see the attachment.

Cheers,
Creeptonight

[mvm.pdf](https://github.com/WeAreMahsaAmini/Core/files/9795779/mvm.pdf)

**Comment by tcf-admin** [comment]
Hi @creeptonight, 

We would like to start a GIS subdomain over https://Iranliberationmovement.com 
We have read your proposal and would love to collaborate to make it a reality.
The website is a collaborative, cooperative and self-reflective forum run by a heterarchical collective, where you can become a member and have an equal share in decision-making.
